Техническое задание №10

ArtemKarpenko403 opened the pull request.

Commit:
Техническое задание №10

Commit:
Добавил unit test

Commit:
ctrl+alt+L

litvitnik commented on src/main/java/ru/yandex/practicum/catsgram/CatsgramApplication.java:
```diff
@@ -0,0 +1,11 @@
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class CatsgramApplication {
```
Этот класс должен называться  FilmorateApplication)

ArtemKarpenko403 replied:
копипса (

litvitnik commented on src/main/java/ru/yandex/practicum/catsgram/controller/UserController.java:
```diff
@@ -0,0 +1,75 @@
+        return existingUser;
+    }
+
+    private void validateUser(User user) {
```
Этот метод не нужен, так как эти условия проверяются аннотациями в классе User)

litvitnik commented on src/main/java/ru/yandex/practicum/catsgram/controller/UserController.java:
```diff
@@ -0,0 +1,75 @@
+        }
+    }
+
+    private void checkEmailUniqueness(String email) {
```
Такого условия нет по ТЗ, этот метод нужно удалить)

Commit:
Исправления.
1. переименовал CatsgramApplication  в FilmorateApplication
2. Убрал private void validateUser(User user)
3. убрал private void checkEmailUniqueness(String email
